setup.py

```diff
@@ -28,7 +28,7 @@ def build_extension(self, ext):
             return
         if sys.platform == "win32" and self.compiler.compiler_type == "mingw32":
             ext.sources.append("source/bases/manifest.rc")
-        os.environ["LD_RUN_PATH"] = "${ORIGIN}:${ORIGIN}/../lib"
+        os.environ["LD_RUN_PATH"] = "${ORIGIN}:${ORIGIN}/lib"
```

Comment:
The original code was intended so that if you installed the executable in /usr/bin (for example) that you would then find the library in /usr/lib. This would eliminate that and require that the files be found in /usr/bin/lib -- which seems wrong! Perhaps it might be satisfactory to simply add ${ORIGIN}/lib and not replace ${ORIGIN}/../lib"?

Reply:
I had been a little doubtful about this point, but some change here is needed. I tested it on ubuntu and alpine and it seemed correct. I didn't understand why it had this "$ {ORIGIN} /../ lib". And I think this "$ {ORIGIN}" could be removed yes. I'll try.

Reply:
I changed it to:
os.environ["LD_RUN_PATH"] = "${ORIGIN}/../lib:${ORIGIN}/lib"

The test pass on alpine linux (with this patch and other that it depend #494)
The test pass on my projects.

Reply:
PR updated.